src/base/http/server.h

```diff
@@ -53,6 +53,7 @@ namespace Http
 
     private slots:
         void dropTimedOutConnection();
+        void removeConnection(Connection *connection);
```

Comment:
I would place it in "private" section below.

Reply:
oops, done.